world_universities_and_domains.json

```diff
@@ -129113,5 +129128,451 @@
+    ],
+    "name": "Institut Superieure de mathematiques et informatique de Monastir",
+    "alpha_two_code": "TN",
+    "state-province": null,
+    "domains": [
+      "isimm.u-monastir.tn"
+    ],
+    "country": "Tunisia"
+  },
+  {
+    "web_pages": [
+      "www.scheppers-wetteren.be"
+    ],
+    "name": "Scheppersinstituut",
+    "alpha_two_code": "BG",
+    "state-province": null,
+    "domains": [
+      "scheppers-wetteren.be"
+    ],
+    "country": "Belgium"
+  },
+  {
+    "web_pages": [
+      "https://immac.fr"
+    ],
+    "name": "LGT Immaculee Conception",
+    "alpha_two_code": "FR",
+    "state-province": null,
+    "domains": [
+      "immac.fr"
+    ],
+    "country": "France"
+  },
+  {
+    "web_pages": [
+      "https://nung.edu.ua/"
+    ],
+    "name": "Ivano-Frankivsk National Technical University of Oil and Gas",
+    "alpha_two_code": "UA",
+    "state-province": null,
+    "domains": [
+      "nung.edu.ua"
+    ],
+    "country": "Ukraine"
+  },
+  {
+    "web_pages": [
+      "bssm.hs.kr/"
+    ],
+    "name": "Busan Software Mester High School",
+    "alpha_two_code": "KR",
+    "state-province": null,
+    "domains": [
+      "bssm.hs.kr"
+    ],
+    "country": "Korea"
+  },
+  {
+    "web_pages": [
+      "https://www.uiz.ac.ma/"
+    ],
+    "name": "Universite Ibn Zohr",
+    "alpha_two_code": "MA",
+    "state-province": null,
+    "domains": [
+      "edu.uiz.ac.ma"
+    ],
+    "country": "Morocco"
+  },
+  {
+    "web_pages": [
+      "https://www.kushiro-ct.ac.jp"
+    ],
+    "name": "National Institute of Technology (KOSEN), Kushiro College",
+    "alpha_two_code": "JP",
+    "state-province": null,
+    "domains": [
+      "kushiro.kosen-ac.jp"
+    ],
+    "country": "Japan"
+  },
+  {
+    "web_pages": [
+      "https://www.htl-hl.ac.at/"
+    ],
+    "name": "Hohere Technische Lehranstalt",
+    "alpha_two_code": "AT",
+    "state-province": null,
+    "domains": [
+      "el.htlhl.at"
+    ],
+    "country": "Austria"
+  },
+  {
+    "web_pages": [
+      "https://ifnmg.edu.br"
+    ],
+    "name": "Instituto Federal do Norte De Minas Gerais",
+    "alpha_two_code": "BR",
+    "state-province": null,
+    "domains": [
+      "aluno.ifnmg.edu.br"
+    ],
+    "country": "Brazil"
+  },
+  {
+    "web_pages": [
+      "www.nagano-nct.ac.jp"
+    ],
+    "name": "National Institute of Technology, Nagano College",
+    "alpha_two_code": "JP",
+    "state-province": null,
+    "domains": [
+      "nagano-nct.ac.jp"
+    ],
+    "country": "Japan"
+  },
+  {
+    "web_pages": [
+      "www.supinfo.com"
+    ],
+    "name": "Supinfo",
+    "alpha_two_code": "FR",
+    "state-province": null,
+    "domains": [
+      "supinfo.com"
+    ],
+    "country": "France"
+  },
+  {
+    "web_pages": [
+      "https://www.klu.edu.tr/"
+    ],
+    "name": "Kirklareli univeritesi",
+    "alpha_two_code": "TR",
+    "state-province": null,
+    "domains": [
+      "ogr.klu.edu.tr"
+    ],
+    "country": "Turkey "
```

Comment:
Probably will be trimmed at some point (or not sure if even used anywhere), but here's an extra space.